### ISSUE TITLE
Allow external info links with href

### ISSUE THREAD

**Issue by shaunix** [projectmallard/projectmallard.org#30]
Currently, any info links have to be xref links to other pages or sections within the document (or potentially in a larger help system with an extended xref syntax). This goes for topic, guide, seealso, and next links, as well as any other links defined by extensions or future Mallard versions.

Sometimes it's useful to link to an external page within topic links or seealso links. We could allow this by allowing an href attribute on info link elements. But then we need a way to get information like title, desc, thumb, etc. So we could allow that information as child elements to the link element.

```
<info>
  <link type="topic" href="http://projectmallard.org/">
    <title>Mallard</title>
    <desc>We built the help system.</desc>
  </link>
</info>
```

Basically, the contents of the link element are treated as if they're the contents of an info element for the thing you're linking to. This allows stuff like sort and text titles, revision info, etc. The current content model for info link elements is just external. Switching it to info contents, which allows external, shouldn't present any compatibility issues.

**Comment by shaunix** [comment]
http://projectmallard.org/mep/mep0007
